tests/fetch/to_memory_pthread.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include <string.h>
+#include <stdio.h>
+#include <math.h>
+#include <assert.h>
+#include <pthread.h>
+#include <emscripten/fetch.h>
+
+int result = 0;
+
+void ThreadMainLoop()
+{
+  printf("TML!\n");
+}
+
+void *ThreadMain(void *arg)
```

Comment:
This function has void* return type, but doesn't return anything. Should probably just return nullptr.